pglast/printers/ddl.py

```diff
@@ -7,14 +7,33 @@
 #
 
 from .. import enums
-from ..node import Missing, List
+from ..node import Missing, List, Node
 from ..printer import node_printer
+import re
+
+
+RE_DOLLARQUOTE = re.compile(r"\$(\w*)\$")
+
+
+
```

Comment:
Please, remove one empty line here: per PEP8, there should be two empty lines between top level functions / global symbols.